src/gui/editors/PianoRoll.cpp

```diff
@@ -342,7 +342,7 @@ PianoRoll::PianoRoll() :
 	// Set up note length model
 	m_noteLenModel.addItem( tr( "Last note" ),
 					std::make_unique<PixmapLoader>( "edit_draw" ) );
-	const auto pixmaps = std::array<QString, 11>{"whole", "half", "quarter", "eighth",
+	const auto pixmaps = std::array<std::string, 11>{"whole", "half", "quarter", "eighth",
```

Comment:
Since these are just string literals, can't we use `std::string_view` here?

Reply:
Just below this, we have these lines:
```cpp
auto loader = std::make_unique<PixmapLoader>( "note_" + pixmaps[i] );
// ...
auto loader = std::make_unique<PixmapLoader>( "note_" + pixmaps[i+NUM_EVEN_LENGTHS] );
```
There's no `operator+(const char*, std::string_view)`, so one of the operands needs to be a `std::string`. We could instead write `"note_"s`, then concatenate onto that, although `pixmaps` would have to be an array of `const char*`, since `operator+(std::string, std::string_view)` doesn't exist until C++26. However, I don't think either way has any particular advantage over the other. I'm inclined to leave it this way, as it's a more minimal change from the original.

Reply:
Oh okay, didn't see this piece of code in the review. 👍